### ISSUE TITLE
[Portuguese] Localize 'Horizontal Scaling'

### ISSUE THREAD

**Issue by edsoncelio** [cncf/glossary#824]
Localize https://glossary.cncf.io/horizontal_scaling/

**Comment by MrErlison** [comment]
New link https://glossary.cncf.io/horizontal-scaling/

**Comment by rafaelmnatali** [comment]
@edsoncelio  @MrErlison  if this still needs to be done, please assign it to me. 

Thanks!

**Comment by edsoncelio** [comment]
> https://glossary.cncf.io/horizontal-scaling/

Hi @rafaelmnatali! 
Let me know If you still want to work on it!

**Comment by rafaelmnatali** [comment]
yes @edsoncelio, please assign to me and will start working on it.

thanks!

**Comment by edsoncelio** [comment]
Closing due to lack of contribution; if someone wants to, we can open a new one.
